### PR TITLE
Statistics integration: Add state characteristic docu

### PR DESCRIPTION
**Pull request by ThomDietrich**
## Proposed change

Hey all,
this PR contains
1. all changes following the addition to the statistics integration https://github.com/home-assistant/core/pull/49960
2. a general rework of the details given about the various characteristics

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/49960
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed not not revived

**Comment by emontnemery** [comment]
Reopening since the core PR is revived

**Comment by emontnemery** [comment]
Please fix the merge conflict.
~~Does this PR include relevant changes also from https://github.com/home-assistant/core/pull/59867?~~
No, it doesn't.

File: source/_integrations/statistics.markdown [hunk 1]
```diff
@@ -14,47 +14,83 @@ ha_platforms:
...
+The created sensor presents one characteristic as its state (the `mean` value by default) and many more as attributes. The time period and/or number of recent state changes to be considered is defined as part of the sensor configuration. The sensor and its characteristics update with every source sensor update.
+
+Both `sensor` and `binary_sensor` are supported as source sensor. In the case of a binary sensor only the state changes are counted.
+
+Assuming the [`recorder`](/integrations/recorder/) integration is running, historical sensor data is read from the database on startup and is available immediately after a restart of the platform. If the [`recorder`](/integrations/recorder/) integration is *not* running, it can take some time for the sensor to start reporting data because some characteristics calculations require more than one source sensor value.
+
+## Characteristics
+
+The following statistical characteristics are provided for the collected source sensor. Pay close attention to the right configuration of `sampling_size` and `max_age`, as most characterists are directly related to the count of samples or the age of processed samples.
```
Comment (frenck) [inline source/_integrations/statistics.markdown]:
```suggestion
The following statistical characteristics are provided for the collected source sensor. Pay close attention to the right configuration of `sampling_size` and `max_age`, as most characteristics are directly related to the count of samples or the age of processed samples.
```

File: source/_integrations/statistics.markdown [hunk 2]
```diff
@@ -14,47 +14,83 @@ ha_platforms:
...
 
 ```yaml
-# enable the recorder integration (optional)
-recorder:
-
-# Example configuration.yaml entry
 sensor:
   - platform: statistics
-    entity_id: sensor.cpu
+    name: Bathroom humidity mean over last 24 hours
```
Comment (frenck) [inline source/_integrations/statistics.markdown]:
```suggestion
    name: "Bathroom humidity mean over last 24 hours"
```

File: source/_integrations/statistics.markdown [hunk 3]
```diff
@@ -14,47 +14,83 @@ ha_platforms:
...
+    entity_id: sensor.bathroom_humidity
+    max_age:
+      hours: 24
+  - platform: statistics
+    name: Bathroom humidity change over 5 minutes
```
Comment (frenck) [inline source/_integrations/statistics.markdown]:
```suggestion
    name: "Bathroom humidity change over 5 minutes"
```

**Review by frenck — APPROVED** [review]
Thanks, @ThomDietrich 👍

**Pull request merged**